drivers/bluetooth/hci/h4.c

```diff
@@ -191,7 +255,7 @@ static void rx_thread(void *p1, void *p2, void *p3)
 		 * original command buffer (if available).
```

Comment:
Does this comment need updating now that  there is a separate pool for num complete?

Reply:
I'm not sure. To be honest, I don't fully understand how the host stack works.